python/huggingface_server.Dockerfile

```diff
@@ -50,6 +50,8 @@ COPY --from=builder --chown=kserve:kserve $VIRTUAL_ENV $VIRTUAL_ENV
 COPY --from=builder kserve kserve
 COPY --from=builder huggingfaceserver huggingfaceserver
 
+# Set a writable cache folder to avoid permission issue. See https://github.com/kserve/kserve/issues/3562
+ENV HUGGINGFACE_HUB_CACHE="/tmp/huggingface/hub"
```

Comment:
Can people still overwrite on the inference service yaml ?

Reply:
Yes I think so.

Reply:
instead of setting [HF_HUB_CACHE](https://huggingface.co/docs/huggingface_hub/en/package_reference/environment_variables#hfassetscache) (Defaults to "$HF_HOME/hub"), can we set [HF_HOME](https://huggingface.co/docs/huggingface_hub/en/package_reference/environment_variables#hfhome) so that other data like `token` can be stored as well.
> To configure where huggingface_hub will locally store data. In particular, your token and the cache will be stored in this folder.

Reply:
Great idea. Done.